proto/select.proto

```diff
@@ -108,13 +108,21 @@ message DAGRequest {
 
 	// Represents the maximum size of one packet, any generated string, or any parameter sent as long data.
 	optional uint64 max_allowed_packet = 13;
+
+	// Represents the tidb system endian.
+	optional Endian tidb_system_endian = 14 [(gogoproto.nullable) = false];
```

Comment:
Prefer to name it as something like `message chunk_memory_layout { endianness }` for two reason:

1. Clearly indicate it is only a setting for the chunk encoding, not the default encoding.
2. Allow the possibility to handle bit width in future.